### PR TITLE
Release notes 2016-08-02

### PR DESCRIPTION
**Pull request by orangejulius**
Wow! So many changes!

Obviously we won't merge this until after we release tomorrow :)

### CODE REVIEW

**Comment by dianashk** [comment]
LGTM Many changes indeed! We've been busy. 🎉 🎈

File: release-notes.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+* Searching for [only records in certain layers in Geonames](https://github.com/pelias/api/pull/573) now works! We keep adding better handling of Geonames data but sometimes our API code doesn't keep up with those changes.
+* Labels now [include county names](https://github.com/pelias/api/pull/576) if there's no city (locality) info present. This helps with [addresses that are outside the bounds of any city](https://github.com/pelias/api/issues/575)
+* Capitalization across all Openaddresess records is now more consistent. We've tried to [properly capitalize all records that were either in all caps, or all lowercase](https://github.com/pelias/openaddresses/pull/131). This is better in general, although there are certainly exceptions, and we welcome bug reports for those cases.
+* Geonames recors for New York City boroughs like Mahnattan and Brooklyn are [now in the `borough`, rather than `locality` layer](https://github.com/pelias/geonames/pull/100). This makes them consistent with the records from Who's on First, which have been boroughs for some time.
```
Comment (trescube) [inline release-notes.md]: `recors` -> `records`

Comment (trescube) [inline release-notes.md]: `Mahnattan` -> `Manhattan`

File: release-notes.md [hunk 2]
```diff
@@ -1,3 +1,21 @@
+## 2 August 2016
+
+Another bigger than usual release, we had some ops related challenges to resolve after the update to Elasticsearch 2, as well as some data issues, but we also have some great improvements in store!
+
+* We use more of the population data in Who's on First, which really helps more [relevant cities come up in searches](https://github.com/pelias/whosonfirst/pull/116).
+* Searching for [only records in certain layers in Geonames](https://github.com/pelias/api/pull/573) now works! We keep adding better handling of Geonames data but sometimes our API code doesn't keep up with those changes.
+* Labels now [include county names](https://github.com/pelias/api/pull/576) if there's no city (locality) info present. This helps with [addresses that are outside the bounds of any city](https://github.com/pelias/api/issues/575)
+* Capitalization across all Openaddresess records is now more consistent. We've tried to [properly capitalize all records that were either in all caps, or all lowercase](https://github.com/pelias/openaddresses/pull/131). This is better in general, although there are certainly exceptions, and we welcome bug reports for those cases.
```
Comment (trescube) [inline release-notes.md]: `Openaddresess` -> `OpenAddresses`

Comment (trescube) [inline release-notes.md]: `caps, or all` -> `caps or all`

File: release-notes.md [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+* Geonames recors for New York City boroughs like Mahnattan and Brooklyn are [now in the `borough`, rather than `locality` layer](https://github.com/pelias/geonames/pull/100). This makes them consistent with the records from Who's on First, which have been boroughs for some time.
+* Addresses in the [Czech Republic](https://github.com/pelias/api/pull/594) now show the streetname before the housenumber, in keeping with local customs
```
Comment (trescube) [inline release-notes.md]: `streetname` -> `street name`

Comment (trescube) [inline release-notes.md]: `housenumber` -> `house number`

File: release-notes.md [hunk 4]
```diff
@@ -1,3 +1,21 @@
...
+* When using the `/v1/place` endpoint, the source name can either be the full name, or the [abbreviation](https://github.com/pelias/api/pull/574) (like the `sources` parameter to the search and autocomplete endpoints). We love saving people some typing :)
```
Comment (trescube) [inline release-notes.md]: `name, or the` -> `name or the` (remove comma)

**Comment by trescube** [comment]
:shipit:

**Pull request merged**